dept44-formatting-plugin/pom.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<modelVersion>4.0.0</modelVersion>
+<groupId>se.sundsvall.dept44</groupId>
+<!--- To avoid circular dependencies, this plugin is not using the parent pom. -->
+<artifactId>dept44-formatting-plugin</artifactId>
+<version>6.0.3-SNAPSHOT</version>
+<packaging>maven-plugin</packaging>
+<properties>
+	<maven.compiler.source>21</maven.compiler.source>
```

Comment:
Isn't these inherited from parent?

Reply:
see row 5. `<!--- To avoid circular dependencies, this plugin is not using the parent pom. -->`